src/test/munit/flow-routing/flow-routing-test-suite.xml

```diff
@@ -50,6 +50,33 @@
     </munit:validation>
   </munit:test>
 
+  <munit:test name="unsupported-media-type-test">
```

Comment:
I think you can move this to "default-error-handler-test-suite" please check munit/error/default-error-handler-test-case.xml